generators/app/index.js

```diff
@@ -156,9 +156,14 @@ module.exports = class extends BaseGenerator {
             type: String
         });
 
-        // This adds support for a `--blueprint` flag which can be used to specify a blueprint to use for generation
+        // NOTE: Deprecated!!! Use --blueprints instead
         this.option('blueprint', {
-            desc: 'Specify a generator blueprint to use for the sub generators',
+            desc: 'Specify one or more generator blueprints to use for the sub generators',
+            type: String
+        });
+        // This adds support for a `--blueprints` flag which can be used to specify one or more blueprints to use for generation
+        this.option('blueprints', {
+            desc: 'Specify one or more generator blueprints to use for the sub generators',
```

Comment:
We should add an example here, also I think yeomen have a List type if I'm not wrong

Reply:
Will add the example. There is indeed an `Array` type that will consume all remaining arguments. See https://yeoman.io/authoring/user-interactions.html. Should I change it?

Reply:
Reworded and added example. Let me know if you wish me to change the type as well

Reply:
I think using array type would be cleaner

Reply:
OK. Will get to it. Will also need to rebase after merge of #9918 tom ake sure that changes in the upgrade gen are also tested. Would it be ok to squash intermediate commits to keep the history cleaner or would that mess up the discussions, etc?

Reply:
you can squash if you like

Reply:
Apparently I misread the docs :man_facepalming:  . The `Array` type is supported for arguments but not options, hence back to the comma separated list. In any case, I have made a few more fixes and code improvements(mainly remove code duplication).